File: 20495_corejava/Day2Assignments/Product.java
package com.sonata;

public class Product {
	
	int proId;
	String proName;
	double proPrice;
	
	Product(){}
	
	Product(int id,String name,double price){
		this.proId=id;
		this.proName=name;
		this.proPrice=price;
	}
	public void CalGst(double price) {
		
		this.proPrice=this.proPrice+price;
		System.out.println(proPrice);
	}
	
	public void dispaly() {
		System.out.println(proId);
		System.out.println(proName);
		System.out.println(proPrice);
	}
	
	public static void main(String [] args) {
		Product p1=new Product();
		p1.proId=1;
		p1.proName="gfd";
		//p1.proPrice=100;
		p1.dispaly();
	}
}

File: TypeScriptAssignment/Product.js
var Product = /** @class */ (function () {
    function Product(proId, proName, proPrice) {
        this.ProId = proId;
        this.ProName = proName;
        this.ProPrice = proPrice;
    }
    Product.prototype.gst = function (a) {
        this.ProPrice = this.ProPrice + (a / 100 * this.ProPrice);
    };
    Product.prototype.display = function () {
        console.log(this.ProId);
        console.log(this.ProName);
        console.log(this.ProPrice);
    };
    return Product;
}());
var p = new Product(1, "Mobile", 20000);
p.gst(10);
p.display();
